.github/workflows/CI.yml

```diff
@@ -226,23 +213,14 @@ jobs:
       - name: Build Windows
         shell: msys2 {0}
         run: |
+          mkdir artifacts
+          
           mkdir sunshine-windows-build && cd sunshine-windows-build
           cmake -DCMAKE_BUILD_TYPE=Release -DSUNSHINE_ASSETS_DIR=assets -G "MinGW Makefiles" ..
           mingw32-make -j2
-      - name: Package Windows
-        run: |
-          cd sunshine-windows-build
-          del ..\assets\apps_linux.json
-          7z a sunshine-windows.zip ..\assets
-          7z a sunshine-windows.zip sunshine.exe
-          7z a sunshine-windows.zip tools\dxgi-info.exe
-          7z a sunshine-windows.zip tools\audio-info.exe
-          7z a sunshine-windows.zip tools\sunshinesvc.exe
-          7z a sunshine-windows.zip ..\tools\install-service.bat
-          7z a sunshine-windows.zip ..\tools\uninstall-service.bat
-          cd ..
-          mkdir artifacts
-          move "sunshine-windows-build\sunshine-windows.zip" "artifacts"
+          cpack
+          
+          cp Sunshine__.exe ../artifacts
```

Comment:
Let's keep the package step as we discussed. Would be good to have a portable version as well.

Probably doesn't make much difference, but I think it would be better (slightly faster) to use `move`... also let's rename the file to `sunshine-windows-installer.exe`

Probably makes most sense to have both `cpack` and the `move` in the package step.

```yaml
      - name: Package Windows
        run: |
          cd sunshine-windows-build

          cpack

          del ..\assets\apps_linux.json
          7z a sunshine-windows.zip ..\assets
          7z a sunshine-windows.zip sunshine.exe
          7z a sunshine-windows.zip tools\dxgi-info.exe
          7z a sunshine-windows.zip tools\audio-info.exe
          7z a sunshine-windows.zip tools\sunshinesvc.exe
          7z a sunshine-windows.zip ..\tools\install-service.bat
          7z a sunshine-windows.zip ..\tools\uninstall-service.bat

          mkdir artifacts
          move "sunshine-windows.zip" "../artifacts"
          move "Sunshine__.exe" "../artifacts/sunshine-windows-installer.exe"
```